vendor/github.com/ahmetalpbalkan/go-httpbin/handlers.go

```diff
@@ -7,11 +7,12 @@ import (
 	"fmt"
```

Comment:
Why are vendor packages being changed? It is a result of syncing up the checked in code with the source repo of the package?